### PR TITLE
issue#13 and issue#3 

### PR DESCRIPTION
**Pull request by SohailAlvi**
#### Checklist :

  - [ ] I have added the relevant documentation.
  - [x] My branch is up-to-date with the upstream master branch.
  - [x] I have fully tested the changes locally.

#### Development environment

    OS: Ubuntu 18.04
    Python Version: 3.5.2

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 bs4==0.0.1
 requests==2.20.1
-scapy==2.4.0
```
Comment (abhisharma404) [inline requirements.txt]: Why!?

**Comment by abhisharma404** [comment]
Update your branch and try to solve as much Codacy issues that can be resolved.

File: requirements.txt (hunk 1 above)
Comment (abhisharma404) [reply requirements.txt]: Correct this.

**Comment by abhisharma404** [comment]
@SohailAlvi Do update your branch.

File: requirements.txt (hunk 1 above)
Comment (SohailAlvi) [reply requirements.txt]: Corrected now. I deleted it by mistake,got confused between scapy and scrapy.

**Comment by abhisharma404** [comment]
@SohailAlvi rebase your branch

**Comment by abhisharma404** [comment]
Resolve your branch conflicts, I'll merge then.

**Pull request merged**